src/debug.c

```diff
@@ -1917,6 +1930,10 @@ void logCurrentClient(client *cc, const char *title) {
     sdsfree(client);
     serverLog(LL_WARNING|LL_RAW,"argc: '%d'\n", cc->argc);
     for (j = 0; j < cc->argc; j++) {
+        if (j >= clientArgsToLog(cc)){
+            serverLog(LL_WARNING|LL_RAW,"client->argv[%d]: *redacted*\n", j);
```

Comment:
corresponds to the following `argv[%d]`.
```suggestion
            serverLog(LL_WARNING|LL_RAW,"argv[%d]: *redacted*\n", j);
```